### ISSUE TITLE
Example code not working

### ISSUE THREAD

**Issue by niederle** [sameeul/I2K2023_Workshop#1]
Hello,
thank you for providing example code with example data for nyxus. I followed the installation instructions and tried to execute the example notebook but already got the following error message with the first attempt to run nyxus:
![error](https://github.com/sameeul/I2K2023_Workshop/assets/740926/109d0429-4a37-4fe5-a557-a45d6fd1ad1e)
Even if I set `single_roi` to `False`, I get another error message (complaining about intensities expected at range 0.0-1.0). It seems there have been changes since that workshop. I cannot get it running.
This is my conda environment:
[conda_env.txt](https://github.com/sameeul/I2K2023_Workshop/files/13582862/conda_env.txt)

**Comment by sameeul** [comment]
Thanks for reporting it. We have recently made some API updated based on the requests from our internal users. Let me take a look at it.

**Comment by sameeul** [comment]
Quick fix: Update calling `featurize_files` method like the following:
```
nyx.set_params(min_intensity=0, max_intensity=10000, dynamic_range=10000)
feature_vals = nyx.featurize_files(int_image_path, seg_image_path, False)
```

TLDR:
So, there are couple of API changes that happened, which made the code fail. Let me try to give a brief explanation for the changes.
We added `single_roi` option, so that user can choose whether they are operating in a whole-slide more or not.
Then we also found that, there can be float data which can have an arbitrary range that later need to be mapped to an integer range. By default, for images with float data, we assume the data range is between 0.0-1.0 and then they are mapped to a range of 0-10000. 

In our test data for the workshop, the segmentation files are actually of float type, hence without providing the mapping, the file reader complains and error out. So, a fix will be to let `Nyxus` know how the mapping should work. Since these are segmentation images, the values only represent ROI ID and we can set the min_intensity=0, max_intensity=10000, dynamic_range=10000 without influencing the computed values. In an ideal case, these segmentation files should only have integer values but unfortunately, the ROI IDs are in float. As I am writing this, I can see that the names of these parameters can be improved. But, for this current version of `Nyxus`, we can simply do `Nyxus.set_params(min_intensity=0, max_intensity=10000, dynamic_range=10000)`

I tested and the following codeblock works.
```
from nyxus import Nyxus
int_image_path = ["./data/evican/intensity/eval_100_SKBR_ch2.ome.tif"]         
seg_image_path = ["./data/evican/label/eval_100_SKBR_ch2.ome.tif"]
nyx = Nyxus(["MIN", "MAX", "MEAN", "CIRCULARITY"])
nyx.set_params(min_intensity=0, max_intensity=10000, dynamic_range=10000)
feature_vals = nyx.featurize_files(int_image_path, seg_image_path, False)
feature_vals.head()
```

**Comment by niederle** [comment]
Thanks a lot for the quick response and the detailed explanation of what's going on!
I can confirm that your proposal works. This helps a lot to play around and investigate the possibilities with nyxus!

**Comment by sameeul** [comment]
The latest docs are here : https://nyxus.readthedocs.io/en/latest/
We have made some API changes on how to write Arrow files. I will try to update the notebook soon to reflect them. Meanwhile, if you have questions, reach out here or https://github.com/PolusAI/nyxus
